### ISSUE TITLE
Selected Project text in Left pane is not having 4.5:1 color contrast ratio .

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#274]
### Steps to Reproduce

1. Launch Xamrin Workbook application. 
2.Create a new workbook and observe the colour contrast ratio of Selected Project in left pane using colour contrast analyser 

### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is lesser than 4.5:1 (i.e., 3.8:1)
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File

![issue 3](https://user-images.githubusercontent.com/22956130/39861644-c7970334-5430-11e8-935a-bc907cce9a9b.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS control. We do not influence or control the rendering of this control. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1831
